prime-router/src/main/kotlin/azure/SendFunction.kt

```diff
@@ -36,14 +38,21 @@ class SendFunction(private val workflowEngine: WorkflowEngine = WorkflowEngine()
     @FunctionName(send)
     @StorageAccount("AzureWebJobsStorage")
     fun run(
-        @QueueTrigger(name = "msg", queueName = send)
-        message: String,
+        @QueueTrigger(name = "msg", queueName = send) message: String,
         context: ExecutionContext,
+        @BindingName("Id") messageId: String? = null,
+        @BindingName("DequeueCount") dequeueCount: Int? = null,
+        @BindingName("NextVisibleTime") nextVisibleTime: Date? = null,
+        @BindingName("InsertionTime") insertionTime: Date? = null,
     ) {
```

Comment:
Azure does not make it easy to pull out metadata about the queue message that triggered the function run.

Reply:
No they do not